### ISSUE TITLE
Allow repeated notification notification messages (F-Droid)

### ISSUE THREAD

**Issue by YamatoRyou** [vector-im/element-android#5004]
#### What would you like to do?
In the case where there is a notification on the screen, and the application is completely exited (including crash), then push again after the next startup (also includes boot automatically starting). Also allow setting intervals for a certain time repeated push notification.

#### Why would you like to do it?
This may be used to resolve: After the application is restarted (including manually restarting because the application crashes, or starts automatically), there is an unread message but does not push a notification.

#### How would you like to achieve it?
n/a

### Have you considered any alternatives?

n/a

### Additional context

Reproduce the steps:
1. A sends a message to B;
2. At this point B receives the push notification of this message, but do not clear the notification;
3. B forcibly stop the application (simulation application crash), at this time, the notice will disappear;
4. B restart the app (or because the application caused by restarting the phone is automatically activated). Enter the application still has unread messages, but there is no push notification.

**Comment by ouchadam** [comment]
the app is persisting the notifications each time we receive a new one for this use case! it sounds like we've got a bug...

out of interest, do the previous notifications display after receiving a new message after a crash?

**Comment by YamatoRyou** [comment]
> the app is persisting the notifications each time we receive a new one for this use case! it sounds like we've got a bug...
> 
> out of interest, do the previous notifications display after receiving a new message after a crash?

No, when the app is restarted, you will only receive notifications "while the app is stopped and after the app is started" (I mean the notifications located in the system notification area).
Notifications that once appeared in the system notification area (not manually set as read or manually cleared) do not reappear since the app was restarted. Unless the other party repeatedly sends me a message that does not meet the above criteria. This seems to simulate "Repeat notification" behavior.
